docs/constants.md

```diff
@@ -52,12 +52,15 @@ title: Constants
 
 ### Attendee types
 * `users` - Logged-in users
+* `groups` - Groups
 * `guests` - Guest without a login
 * `emails` - A guest invited by email address
 
 ### Actor types of chat messages
 * `guests` - guest users
 * `users` - logged-in users
+* `emails` - users invited by email
+* `groups` - groups
```

Comment:
actually groups can't be author of chat messages.

Reply:
was this a mistake ? https://github.com/nextcloud/spreed/blob/master/src/constants.js#L54

Reply:
Attendees can have such actor types, but not chat messages.

Reply:
ah, got it. I never realized that actor type constants were used in two places, I'll remove that line

Reply:
https://github.com/nextcloud/spreed/pull/5722